zcutil/clean-help-dev.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# If `make clean` fails for any reason, it is helpful to
+# manually clean any remaining files from the previous build.
+# This is especially useful if the new build is for a different
+# architecture than the previous build, such as a Windows release
+# after an OSX cross-compile. Normally these files should be
+# cleaned during `make clean` execution.
+
+is_git_root=0
+
+function check_git_root() {
+
+  local dir=$(pwd)
+
+  while [[ "$dir" != "" && ! -d "$dir/.git" ]]; do
+    dir=${dir%/*} # ${parameter%word}
+  done
+
+  if [ "$dir" == "" ]; then
+    echo "$(pwd) is not in a Git repository."
```

Comment:
add `Exiting...` to communicate negative outcome user feedback

Reply:
[Fixed](https://github.com/KomodoPlatform/komodo/pull/645/commits/694e89738962c867def0750e0c415f237b4f2cdb). Let's approve and merge it first, since all other PRs go through the Windows build, and it fails because this one hasn't been accepted yet.